e2e/test/config/Configuration.AzureSecurityCenterForIoTLogAnalytics.cs

```diff
@@ -8,7 +8,7 @@ public static partial class Configuration
         public static class AzureSecurityCenterForIoTLogAnalytics
         {
             // The Azure Active directory tenant (The subscription tenant)
-            public static string AadTenant => GetValue("LA_AAD_TENANT");
+            public static string AadTenant => GetValue("MSFT_TENANT_ID");
```

Comment:
Are we now using/ planning to use this AAD app to do more than auth for log analytics? If yes, then a comment would help here, listing the operations where this AAD app is now used (for debugging purposes).

Reply:
Oh, disregard the above comment. Scrolling down I see this is hardcoded to be the msft tenant.